### ISSUE TITLE
WebXR - VR

### ISSUE THREAD

**Issue by sylwesterdigital** [croquet/microverse#25]
Am I missing something or this project is not working in WebXR? Any chance to explore play with this using a head mounted VR head set?

**Comment by codefrau** [comment]
Correct, we do not have a WebXR renderer yet. There was a discussion on our [Discord](https://croquet.io/discord) server though, someone (with the user handle `Physicslibrary#3162`, if you want to search) is experimenting with that.

**Comment by yoshikiohshima** [comment]
The initial version of WebXR support is in main. It needs some clean up and stabilization.
